docs/ocean/design_docs/vert-lagrang-remap/vert-lagrang-remap.rst

```diff
@@ -0,0 +1,302 @@
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: To the extent that it is feasible, the user should have the ability to control the order of accuracy and other parameters of the remapping (e.g. via namelist options).
+
+Requirement: multiple time stepping schemes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: Remapping should be supported (eventually) in both the RK4 and the split-explicit time stepping schemes. 
+
+Requirement: not climate changing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: For the same target grid, the solution for layer thicknesses and key prognostic variables using VLR is within a non-climate-changing threshold value [presumably greater than machine precision?] of the solution using split-explicit and RK4 after a few timesteps.
+
+Requirement: performance
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: Performance is not degraded below an acceptable level.
+
+Requirement: conservation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: Remapping operation is conservative in the layer-wise sense.
+
+Requirement: no interference
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: No calculations are made outside of remapping itself using prognostic (or diagnostic) variables both before and after remapping to avoid introducing additional errors.
+
+Requirement: modularity
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: To the degree possible, the code for performing remapping should be kept in its own Fortran module(s) for better readability.
+
+*Each requirement is to be listed under a "section" heading, as there will be a
+one-to-one correspondence between requirements, design, proposed implementation
+and testing. Requirements should not discuss technical software issues, but
+rather focus on model capability. To the extent possible, requirements should
+be relatively independent of each other, thus allowing a clean design solution,
+implementation and testing plan.*
+
+
+Algorithm Design (optional)
+---------------------------
+
+Algorithm Design: remapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+The vertical Lagrangian-remap can be characterized by the following equations, as written in Griffies and Adcroft (2020):
+
+Lagrangian layer motion
+.. math::
+   w^{grid}_{k,*} = -D^n_{k,*}
```

Comment:
rst is picky about spacing [(see the documentation)](https://www.sphinx-doc.org/en/1.0/ext/math.html#directive-math)  I think you need some extra white space for this to work out:
```suggestion
Lagrangian layer motion

.. math::

      w^{grid}_{k,*} = -D^n_{k,*}
```
Even so, it probably won't show up in the GitHub preview and you'd need to do a local build of the documentation to see it.  Let me know if you need instructions for doing that.